lib/screens/dateTimePicker.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { StyleSheet, View, Button } from 'react-native';
+import DateTimePickerModal from "react-native-modal-datetime-picker";
+
+export default DateTimePicker = () => {
+    const [date, setDate] = useState(new Date());
```

Comment:
We should probably figure out whether we want to use lifecycle methods, or use hooks throughout the application, since we're using lifecycles for everything else but hooks here. I get that we copied pasted this logic but in the end you're gonna have to end up knowing how to use both to debug this thing so it might be easier to just learn/use one. Just my thoughts, up to you guys on this

Reply:
is the lifecycle method when we create a class with constructor to set states and hooks method creating a function and using useState?? I agree we should keep it consistent but idk which one to use lol which do you guys think is more efficient/better for our purposes??

Reply:
Yes, idk which one is better. Up to the team, we can talk about this

Reply:
changed to class based in latest commit, resolving this change